### PR TITLE
fix(errors): report to sentry actual error

### PR DESCRIPTION
**Pull request by petrjasek**
instead of superdesk errors so we get some idea what is the issue

### CODE REVIEW

**Comment by naspeh** [comment]
I think it's better to use `logging` module for that:
- https://docs.sentry.io/clients/python/integrations/logging/

in our codebase we should use `logger.exception()` instead of `app.sentry.captureException()`

with this I can configure logging to file, to sentry, to logstash, etc...

**Comment by petrjasek** [comment]
good point, will update

File: superdesk/io/feed_parsers/rfc822.py [hunk 1]
```diff
@@ -132,7 +133,8 @@ def parse(self, data, provider=None):
                                 html_body = self.safe_html(html_body)
                                 continue
                             except Exception as ex:
-                                logger.exception(
+                                logger.exception()
+                                logger.error(
```
Comment (naspeh) [inline superdesk/io/feed_parsers/rfc822.py]: Maybe just one error logging with `exc_info=True`, like:
```
# If you're actually catching an exception, use `exc_info=True`
logger.error('There was an error, with a stacktrace!', exc_info=True)

```

**Comment by naspeh** [comment]
Sorry, seems you need `logger.exception(ex)` everywhere

**Comment by petrjasek** [comment]
yep makes sense, working on it

**Comment by naspeh** [comment]
@petrjasek thanks! :+1:

**Comment by mdhaman** [comment]
in most cases, we are doing `logger.exception(ex)` and  then raising some Superdesk defined errors, can we move logger.exception(ex) into Superdesk Errors.

**Comment by petrjasek** [comment]
@mdhaman updated it to log exceptions for publish/ingest/internal errors

**Pull request merged**